### DISCUSSION TITLE
Placing simple 3D model at many coordinates

### DISCUSSION THREAD

**Discussion by hardreddata** [TerriaJS/terriajs#6662] [Q&A]
Hi,

Suppose I have a set of (x,y,z) coordinates in a GIS data file and I want to put a sphere at each of them and display it in Terria.

Is this possible?

For extra credit I might look to have different coloured spheres, perhaps based on data attributes.

Also I am thinking spheres or cubes and not point cloud.

Advice invited.

Thanks.

**Comment by imakihi** [comment]
You can use czmlTmplate in GeoJsonCatalogItem and other 2D data catalogitems. 
https://docs-v8.terria.io/guide/connecting-to-data/catalog-type-details/geojson/
